extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/TableCommands.scala

```diff
@@ -518,6 +518,19 @@ object TableCommands {
     TableCommandSpec(cmd, Nil, queryDescs = Seq(queryDesc))
   }
 
+  val InsertIntoHadoopFsRelationCommand = {
+    val cmd = "org.apache.spark.sql.execution.datasources.InsertIntoHadoopFsRelationCommand"
+    val actionTypeDesc = ActionTypeDesc(null, null, Some("UPDATE"))
```

Comment:
Shall we consider SaveMode？

Reply:
OK, will add a SaveModeActionTypeExtractor to determine ActionType `INSERT` or `INSERT_OVERWRITE` in our case.